desc/compute/_neoclassical.py

```diff
@@ -86,70 +76,41 @@
         Whether to use cubic splines to compute bounce points.
         """,
     "_vander": """dict[str,jnp.ndarray] :
-        Precomputed transform matrices "dct spline", "dct cfl", "dft cfl".
-        This parameter is intended to be used by objectives only.
+        Precomputed transform matrix "dct spline".
+        This private parameter is intended to be used only by
+        developers for objectives.
         """,
+    "theta": "",
 }
 
 
-def _compute(
-    fun,
-    fun_data,
-    data,
-    theta,
-    grid,
-    num_pitch,
-    surf_batch_size=1,
-    simp=False,
-    expand_out=True,
-):
-    """Compute Bounce2D integral quantity with ``fun``.
-
-    Parameters
-    ----------
-    fun : callable
-        Function to compute.
-    fun_data : dict[str, jnp.ndarray]
-        Data to provide to ``fun``. This dict will be modified.
-    data : dict[str, jnp.ndarray]
-        DESC data dict.
-    theta : jnp.ndarray
-        Shape (num rho, X, Y).
-        DESC coordinates θ from ``Bounce2D.compute_theta``.
-        ``X`` and ``Y`` are preferably rounded down to powers of two.
-    grid : Grid
-        Grid that can expand and compress.
-    num_pitch : int
-        Resolution for quadrature over velocity coordinate.
-    surf_batch_size : int
-        Number of flux surfaces with which to compute simultaneously.
-        Default is ``1``.
-    simp : bool
-        Whether to use an open Simpson rule instead of uniform weights.
-    expand_out : bool
-        Whether to expand output to full grid so that the first dimension
-        has size ``grid.num_nodes`` instead of ``grid.num_rho``.
-        Default is True.
+@register_compute_fun(
+    name="V_r/psi_r",
+    label="\\int \\vert B^{\\zeta} \\vert^{-1} \\mathrm{d}\\alpha \\mathrm{d}\\zeta",
+    units="m^{3} / Wb",
+    units_long="cubic meters per Weber",
+    description="Surface integrated volume Jacobian determinant of "
+    " Clebsch field line coordinate system (ψ,α,ζ)"
+    " where ζ is the DESC toroidal coordinate.",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    resolution_requirement="tz",
+    data=["psi_r/sqrt(g)"],
+)
+def _field_line_weight(params, transforms, profiles, data, **kwargs):
+    """∬_Ω abs(𝐁⋅∇ζ)⁻¹ dα dζ where (α,ζ) ∈ Ω = [0, 2π)².
 
+    The returned quantity has shape (num rho, ).
     """
-    for name in Bounce2D.required_names:
-        fun_data[name] = data[name]
-    fun_data.pop("iota", None)
-    for name in fun_data:
-        fun_data[name] = Bounce2D.fourier(Bounce2D.reshape(grid, fun_data[name]))
-    fun_data["iota"] = grid.compress(data["iota"])
-    fun_data["theta"] = theta
-    fun_data["pitch_inv"], fun_data["pitch_inv weight"] = Bounce2D.get_pitch_inv_quad(
-        grid.compress(data["min_tz |B|"]),
-        grid.compress(data["max_tz |B|"]),
-        num_pitch,
-        simp=simp,
+    data["V_r/psi_r"] = surface_integrals(
```

Comment:
We already have compute qtys for `V_r(r)` and `psi_r`, isn't this just the same as `data['V_r(r)'] / data['psi_r'])`

Reply:
anything that differs from the implementation i have written makes it a total pain to do the axis limit